### ISSUE TITLE
w_02 t03_benchmarking wrong formula on slide from KI-Campus

### ISSUE THREAD

**Issue by SwiftPredator** [automl-edu/AutoMLLecture#37]
I think that I found an error on a slide regarding the Two Matched Samples t-Test. 
The mean performance difference d is calculated wrong. It should be divided by the length of the test set.

In the latex file here in repo, however, the formula seems to be correct. 
It could also be that the slides on the website are no longer up to date, but I would still like to point this out.

If I am wrong, please let me know. 

![Bildschirmfoto 2021-04-23 um 20 05 57](https://user-images.githubusercontent.com/20613170/115912280-65bad400-a46f-11eb-8525-b20f9d82adb3.png)

**Comment by larskotthoff** [comment]
That's correct -- the slide you posted is incorrect, but this is correct in the repo. The video unfortunately has the incorrect version of the slide.

**Comment by f2010126** [comment]
@larskotthoff The video has 1/n in the equation (15:35 of video 3). That is the correct equation then?

**Comment by larskotthoff** [comment]
Apologies, I was looking at the wrong video -- yes, the slide in the video is correct.
